operator/internal/controller/redpanda/redpanda_decommission_controller.go

```diff
@@ -148,6 +163,37 @@ func (r *DecommissionReconciler) Reconcile(c context.Context, req ctrl.Request)
 	return result, err
 }
 
+func (r *DecommissionReconciler) GetRedpanda(ctx context.Context, log logr.Logger, releaseName, namespace string) (*v1alpha2.Redpanda, error) {
+	rp := v1alpha2.Redpanda{}
+	if !r.operatorMode {
+		values, err := getHelmValues(log, releaseName, namespace)
+		if err != nil {
+			return nil, fmt.Errorf("getting helm values: %w", err)
+		}
+
+		wrappedValues := map[string]any{
```

Comment:
Will name and namespace be required by anything consuming the returned `Redpanda`?